### ISSUE TITLE
document all known cache types

### ISSUE THREAD

**Issue by GoogleCodeExporter** [opencaching/okapi#220]
```
(issue 210)
> If we decide to expose something via OKAPI, then I think we should try to 
document it as well as we can.

So the other cache types should be documented, at least those which will not be 
discarded in the near future:

* Moving cache (PL, DE)
* Webcam cache (PL, DE)
* Podcast cache (PL)
* Drive-In cache (DE)

```

Original issue reported on code.google.com by `following09` on 14 Apr 2013 at 4:30

**Comment by following5** [comment]
"Unknown" type is also missing in the OKAPI docs, though OKAPI itself returns it for unknown types.

**Comment by following5** [comment]
Done:
* added documentation of Moving and Webcam cache types
* allow to search for types not available at the OC installation, and ignore them

Types to do:
* Other
* Drive-In, Math/Physics (OCDE only)
* Podcast, Own (OCPL only)

**Comment by following5** [comment]
fixed by 3b2ebae1
